changelog/20882.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
```

Comment:
Can we also add a changelog to mention the feature itself as an `release-note:feature`? I believe that you have have it in this same txt file, right under this one.